starfyre/parser.py

```diff
@@ -167,18 +164,23 @@ def handle_endtag(self, tag):
             if len(self.stack) > 0:
                 parent_node = self.stack[-1]      #this is last item/"top element" of stack
 
-                if endtag_node.original_name != "": #this node is for "custom" component, so we should check for <slot> tags 
-                    #process endtag_node children
+                if endtag_node.original_name != endtag_node.tag: #this node is for "custom" component
```

Comment:
@SuelenKarbivnychyy , I thought that we discussed an approach where we decided to not distinguish between custom components and regular components. 

We should not distinguish between custom components and regular components.

Reply:
> We should not distinguish between custom components and regular components.

@sansyrox 
I need to know where to insert the content of `self.current_children` instead of `slot`. I'm not sure what is "regular component". We have "component definition" that is the pyml file named `parent.fyre`, and we have usage of this component in another pyml file (<parent> in the __init__.fyre) - that usage I'm referring to as "custom component", but we don't have "is_custom" property anymore, like we discussed.